### PR TITLE
Zend_File_Transfer: composer safe setAdapter method

### PR DESCRIPTION
**Pull request by glensc**
Fixes: #11

### CODE REVIEW

File: packages/zend-file-transfer/library/Zend/File/Transfer.php [hunk 1]
```diff
@@ -64,18 +63,15 @@ public function __construct($adapter = 'Http', $direction = false, $options = ar
...
     public function setAdapter($adapter, $direction = false, $options = array())
     {
-        if (Zend_Loader::isReadable('Zend/File/Transfer/Adapter/' . ucfirst($adapter). '.php')) {
-            $adapter = 'Zend_File_Transfer_Adapter_' . ucfirst($adapter);
-        }
-
-        if (!class_exists($adapter)) {
-            Zend_Loader::loadClass($adapter);
-        }
+	    if (class_exists('Zend_File_Transfer_Adapter_' . ucfirst($adapter))) {
```
Comment (falkenhawk) [inline packages/zend-file-transfer/library/Zend/File/Transfer.php]: could you adjust the indentation here please?

File: packages/zend-file-transfer/library/Zend/File/Transfer.php [hunk 2]
```diff
@@ -22,7 +22,6 @@
 /**
  * @see Zend_Loader
  */
-// require_once 'Zend/Loader.php';
```
Comment (falkenhawk) [inline packages/zend-file-transfer/library/Zend/File/Transfer.php]: do not remove commented out `require_once`s, please, as they are simply left there to be in all other occurrences - I would not do any code administrative/beautifying/cleaning tasks in the whole framework and keep the code changes only to the really necessary ones.

File: packages/zend-file-transfer/library/Zend/File/Transfer.php (hunk 1 above)
Comment (glensc) [reply packages/zend-file-transfer/library/Zend/File/Transfer.php]: powers of web edit. looks like have to check this out locally. thanks for the reivew.

**Comment by glensc** [comment]
cleaned up now

**Pull request merged**